README.md

```diff
@@ -111,6 +111,17 @@ Related Modules
 
 * [jsonld.js][]: An implementation of the [JSON-LD][] specification.
 
+### Using with React Native
+
+Using this library with React Native requires the
+[`data-integrity-rn`](https://github.com/digitalcredentials/data-integrity-rn)
+polyfill to be imported before this library:
```

Comment:
Can we say that "a polyfill is required, such as" instead? I don't want to create a defacto dependency.

Reply:
@dlongley - done, updated the language.